src/e2e-test/java/io/cdap/plugin/salesforcesink/stepsdesign/DesignTimeSteps.java

```diff
@@ -17,8 +17,36 @@
 
 package io.cdap.plugin.salesforcesink.stepsdesign;
 
+import io.cdap.e2e.pages.actions.CdfPipelineRunAction;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.BQValidation;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.text.ParseException;
+
 /**
  * Design-time steps of Salesforce plugins.
  */
 public class DesignTimeSteps {
+
+  @Then("Validate the values of records transferred to target Salesforce is equal to the values from source " +
+    "BigQuery table")
+  public void validateTheValuesOfRecordsTransferredToTargetSalesforceIsEqualToTheValuesFromSourceBigQueryTable()
+    throws IOException, InterruptedException, IOException, SQLException, ClassNotFoundException, ParseException {
+    int sourceBQRecordsCount = BigQueryClient.countBqQuery(PluginPropertyUtils.pluginProp("bqSourceTable"));
+    BeforeActions.scenario.write("No of Records from source BigQuery table:" + sourceBQRecordsCount);
+    Assert.assertEquals("Out records should match with target PostgreSQL table records count",
```

Comment:
Update the message Postgres is mentioned here.

Reply:
done